### ISSUE TITLE
can't login

### ISSUE THREAD

**Issue by HengeDK** [NNTmux/newznab-tmux#1173]
**Describe the bug**
clean install again again and can't login
just redirect to login page

**Expected behavior**
login to site

**Screenshots**

**Installation (please complete the following information):**
 - OS: Ubuntu 18.04
 - PHP version 7.4
 - NNTmux fresh install

**Additional context**
found info about same problems where redis would fix it
installed redis and changed .env 
CACHE_DRIVER=redis
REDIS_CLIENT=predis

still same problems

**Comment by DariusIII** [comment]
In your .env file enable debugging and enable telescope in production. Set your site to local so debugging will be live. You will see a page with a lot more info to be worked with.

**Comment by DariusIII** [comment]
Also you can open yoursite/telescope and errors can be seen there too

**Comment by HengeDK** [comment]
I am running with debugging and telescope
telescope is just spinning all time

**Comment by DrakeJones** [comment]
As sure as the sun rises, the new crop checks in.  You are a benevolent individual, Darius.  We appreciate it.  Not a dis, @HengeDK .  You're very lucky to have such an elegant piece of software to learn with.

**Comment by DariusIII** [comment]
Can you share the screenshot of login page with debug enabled?

**Comment by HengeDK** [comment]
![nntmux](https://user-images.githubusercontent.com/9641867/128643320-b4bf23e6-8da3-4842-9ef8-5648aaf268c4.png)

APP_ENV=local
APP_DEBUG=true

TELESCOPE_ENABLED=true
TELESCOPE_FULL_IN_PRODUCTION=true
![telescope](https://user-images.githubusercontent.com/9641867/128643392-d8be19f7-9618-48dc-8ad5-0cc7484f0019.png)

telescope just spins forever

really appreciate your help
have send you appreciation on .patreon

**Comment by HengeDK** [comment]
nothing in storage/logs
admin user is added to database

I can create an account on the server and send it to you
if you would have a look

**Comment by DariusIII** [comment]
Have you set your site  environment to local? You should have an error on login page with debug data presented to you.

**Comment by DariusIII** [comment]
> nothing in storage/logs
> admin user is added to database
> 
> I can create an account on the server and send it to you
> if you would have a look

You can send me details privately.

**Comment by Fossil01** [comment]
Run `php artisan config:cache` after changing anything in .env it might help. Also paste the output of the command `php -m` and the webserver error log or php-fpm error log OR there might be a file called php-errors.log (depends on your php-fpm config) in the root of the NNTmux folder.

**Comment by HengeDK** [comment]
Can you send your e-mail to jakobsen55@protonmail.com
I can’t send you private message here

**Comment by HengeDK** [comment]
`
[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
hash
iconv
intl
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
Reflection
session
shmop
SimpleXML
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache

`

**Comment by luser01** [comment]
@HengeDK how did you resolve this?

**Comment by DariusIII** [comment]
Session domain needed to be empty. It was set to localhost. I was the one
that checked his server and fixed the issue.

**Comment by luser01** [comment]
@DariusIII thanks for your response. I am having issues also. session domain is empty. Tried on ubuntu 18.04 and 20.04
Clean installs on both. I Pm'd you on patreon. I haven't used Laravel since 5.7, and from my debugging skills. 
I grabbed a dump, which made it to telescope right before                     
`
return redirect()->intended($this->redirectPath())->with('info', 'You have been logged in');
`
in app/Http/Controllers/Auth/LoginController.php
I also see all the auth updates in queries. 
redis is installed and I have keys. I also see it in telescope.
